File: webapi/Model/Aluno.cs
namespace alunoapi.Model
{
    public class Aluno
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public int Idade { get; set; }
        public string Email { get; set; }
        public string Senha { get; set; }
        public string Cpf { get; set; }
        public string Rg { get; set; }
        public string Genero { get; set; }
        public string Telefone { get; set;}
        public string Endereco { get; set; }
        public string Escolaridade { get; set; }
        public object Data_Nascimento { get; internal set; }
    }
}

File: treetedb/src/main/java/com/model/entity/Aluno.java
package com.model.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "aluno")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Aluno {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column
	private Long id;
	
	@Column
	private String nome;
	
	@Column
	private String email;
	
	@Column
	private String senha;
	
	@Column
	private String idade;
	
	@Column
	private String sexo;
	
	@Column
	private String cpf;
	
	@Column
	private String rg;
	
	@Column
	private String telefone;
	
	@Column
	private String endereco;
	
	@Column
	private String escolaridade;

}	